.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - 2.6
```

Comment:
The 2.6 build has started failing and as pygit2 doesn't officially support 2.6, let's not test against it.